justfile

```diff
@@ -253,17 +253,20 @@ print-conn-str:
 lint: fmt clippy
 
 # Run cargo fmt
-fmt:
+fmt: (install-markdownlint-cli2)
```

Comment:
add a separate `fmt-md` target that runs `docker run ...` command to format the files. Lets not make it part of the `fmt` just yet.

Reply:
Moved now.